components/typography/stories/typography.stories.js

```diff
@@ -115,57 +131,185 @@ WithForcedColors.parameters = {
 };
 
 // ********* DOCS ONLY ********* //
-export const Heading = Template.bind({});
-Heading.tags = ["!dev"];
-Heading.args = {
+
+// ====== Docs: Sizing ====== //
+export const HeadingSizes = (args, context) => Sizes({
+	Template,
+	direction: "column",
+	...args,
+}, context);
+HeadingSizes.tags = ["!dev"];
+HeadingSizes.args = {
 	semantics: "heading",
 	content: ["Aliquet mauris eu"],
 };
-Heading.parameters = {
+HeadingSizes.parameters = {
 	chromatic: { disableSnapshot: true },
 };
 
-export const Body = Template.bind({});
-Body.tags = ["!dev"];
-Body.argTypes = {
+/**
+ * Body is primarily used for Spectrum components and for blocks of text.
+ */
+export const BodySizes = (args, context) => Sizes({
+	Template,
+	direction: "column",
+	...args,
+}, context);
+BodySizes.argTypes = {
 	size: size(["xs", "s", "m", "l", "xl", "xxl", "xxxl"]),
 };
-Body.args = {
+BodySizes.tags = ["!dev"];
+BodySizes.args = {
 	semantics: "body",
 	content: [
 		"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec eleifend est mollis ligula lobortis, tempus ultricies sapien lacinia. Nulla ut turpis velit. Sed finibus dapibus diam et sollicitudin. Phasellus in ipsum nec ante elementum congue eget in leo. Morbi eleifend justo non rutrum venenatis. Fusce cursus et lectus eu facilisis. Ut laoreet felis in magna dignissim feugiat.",
 	],
 };
-Body.parameters = {
+BodySizes.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * When typography elements are paired, such as with heading and body below, clear content hierarchies are shown.
+ * 
+ * Note that the body component is not available in XXS, so pairing with the size XXS heading is not displayed here.
```

Comment:
On the docs site, I see that an XXS heading is also shown paired with XS body text. Is that fine to no longer show? If we're not showing it, would rewording this sentence to something like this be helpful: "Since the body component is not available in size XXS, the size XXS heading can be paired with the size XS body."